frontend/discourse/app/templates/preferences/profile.gjs

```diff
@@ -1,246 +1,396 @@
+        </form.Field>
       {{/if}}
-      {{#if @controller.canUploadUserCardBackground}}
-        <div
-          class="control-group pref-profile-bg"
-          data-setting-name="user-card-bg"
+
+      {{#each @controller.userFields as |uf|}}
+        <form.Field
+          @name={{concat "user_field_" uf.field.id}}
+          @title={{uf.field.name}}
+          @description={{uf.field.description}}
+          @validation={{validationFor uf.field}}
+          as |field|
         >
-          <label class="control-label">{{i18n
-              "user.change_card_background.title"
-            }}</label>
-          <div class="controls">
-            <UppyImageUploader
-              @imageUrl={{@controller.model.card_background_upload_url}}
-              @onUploadDone={{@controller.cardBackgroundUploadDone}}
-              @onUploadDeleted={{fn
-                (mut @controller.model.card_background_upload_url)
-                null
-              }}
-              @type="card_background"
-              @id="profile-card-background-uploader"
-            />
-          </div>
-          <div class="instructions">
-            {{i18n "user.change_card_background.instructions"}}
-          </div>
-        </div>
-      {{/if}}
-    {{/if}}
-
-    {{#if @controller.siteSettings.allow_featured_topic_on_user_profiles}}
-      <div class="control-group" data-setting-name="user-featured-topic">
-        <label class="control-label">{{i18n "user.featured_topic"}}</label>
-        {{#if @controller.model.featured_topic}}
-          <label class="featured-topic-link">
-            <LinkTo
-              @route="topic"
-              @models={{array
-                @controller.model.featured_topic.slug
-                @controller.model.featured_topic.id
-              }}
+          <field.Custom>
+            <MutableField
+              @value={{field.value}}
+              @set={{field.set}}
+              as |wrapper|
             >
-              {{replaceEmoji
-                (htmlSafe @controller.model.featured_topic.fancy_title)
-              }}
-            </LinkTo>
-          </label>
-        {{/if}}
+              <UserField
+                @field={{uf.field}}
+                @value={{wrapper.value}}
+                @validation={{field.validation}}
+                @showLabel={{false}}
+              />
+            </MutableField>
+          </field.Custom>
+        </form.Field>
+      {{/each}}
 
-        <div>
+      {{#if @controller.canChangeBio}}
+        <form.Field
+          @name="bio_raw"
+          @title={{i18n "user.bio"}}
+          @format="full"
+          as |field|
+        >
+          <field.Custom>
+            <MutableField
+              @value={{field.value}}
+              @set={{field.set}}
+              as |wrapper|
+            >
+              <DEditor
+                @value={{wrapper.value}}
+                @forceEditorMode={{USER_OPTION_COMPOSITION_MODES.rich}}
+                @hidePreview={{true}}
```

Comment:
I made some changes here to avoid this:


<img width="324" height="580" alt="image" src="https://github.com/user-attachments/assets/3c8753c0-75ad-4e82-86e4-98f8ebb60612" />

I made it so this always is in Rich editor mode, hides the preview, and made the field itself `@format="full"` so its wider